src/main/java/net/milkbowl/vault/economy/AbstractEconomy.java

```diff
@@ -59,20 +106,45 @@ public EconomyResponse depositPlayer(OfflinePlayer player, String worldName, dou
 		return depositPlayer(player.getName(), worldName, amount);
 	}
 
+	@Override
+	public EconomyResponse depositPlayer(UUID uuid, double amount) {

```

Comment:
Since these methods are no longer player-only, should they be renamed? Of course API compatibility is of concern, although just noting this.

Reply:
A good catch I will be pushing a commit that drops the Player from the new UUID method names.